core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -164,7 +164,11 @@
             ng-mouseleave="onMouseoutDropdownMenu($event)">
           <li>
             <a href style="padding: 0; width: 200px;" ng-click="onLoginButtonClicked()">
-              <img ng-src="<[getStaticImageUrl('/signin/Red-signin-Long-base-44dp.png')]>" alt="Sign in with Google">
+              <div class="oppia-signin-google">
+                <i class="oppia-signin-google-icon fa fa-google-plus"></i>
```

Comment:
For screen readers? The span element? 
 `<span class="oppia-icon-accessibility-label">Sign In with Google</span>`